.eslintrc

```diff
@@ -1,4 +1,7 @@
 {
   "parser": "babel-eslint",
-  "extends": "atom-build"
+  "extends": "atom-build",
+  "rules": {
+    "no-cond-assign": [2, "except-parens"]
```

Comment:
Lets you do assignments inside `if` statements as long as you surround with extra set of parens